python/omnifold.py

```diff
@@ -212,16 +212,13 @@ def omnifold(
 
     return weights_unfold
 
-# evaluate a step in OmniFold method, training a model if needed and train the model
-# weights_push is modified for reweighting
-# Return: None
 def train_step(
     # Data
     X_step, # feature array of observed data, for this step
     Y_step, # label array, for this step
     w_step, # w_array, for this step
-    weights_update, # arraylike, weights_pull or weights_push to be updated
-    weights_update_range, # range of weights_update to be updated, the entire weights_update_range is updated if NONE is supplied
+    weights_multiplier, # arraylike, multiplied by the model reweight to produce the new weight to be returned
```

Comment:
I don't think It is necessary to have `weights_multiplier` as an argument. The only time it is not `None` is at step 1, and the factor can probably be applied on the returned value of the function: `weights_pull = weights_pull * train_step(...)`. Without `weights_multiplier`, the nested ifs for return can also be simplified for better readability.

Reply:
Sure!